source/js/diva.js

```diff
@@ -939,50 +939,30 @@ module.exports = diva;
         };
 
         /*
-            Given a pageX and pageY value (as could be retreived from a jQuery event object),
```

Comment:
In testing, I think I found that the x,y should come from the jQuery event `e.clientX` and `e.clientY` not the `e.pageX` and `e.pageY` where e.client property take into account and scrolling on the page.  Let me do more testing and put a better comment.